tests/clients/web/CMakeLists.txt

```diff
@@ -1,2 +1,2 @@
 # All of our core javascript will be checked with a single test
-add_javascript_style_test(core src)
```

Comment:
I might be missing something @jbeezley, but is this running on project_root/src or project_root/clients/web/src? I picked a random file in clients/web/src and started changing indentation and removing semicolons and everything still passed.